src/styles/base/variables.scss

```diff
@@ -0,0 +1,24 @@
+$main-text-color: #060b35;
+$secondary-text-color: #616070;
+$nav-link-text-color: #060b35;
+$header-box-shadow: #0000000d;
+$card-link-text-color: #fff;
+$card-link-bg-color: #00acdc;
+$card-link-hovered-bg-color: #fff;
+$card-border-color: #f3f3f3;
+$hover-card-tittle-color: #34568b;
+$link-action-text-color: #00acdc;
+$text-font-weight-low: 400;
+$text-font-weight-medium: 500;
+$text-font-weight-high: 700;
+$text-font-size-smallest: 10px;
+$text-font-size-small: 12px;
+$text-font-size-medium: 14px;
+$text-font-size-big: 16px;
+$text-line-height-small: 14px;
+$text-line-height-medium: 16px;
+$text-line-height-hight: 18px;
```

Comment:
don't use simple variables for fonts.
If you want to export all fonts to a separate file and give them variables, use the following approach
<img width="601" alt="Знімок екрана 2024-08-16 о 14 52 47" src="https://github.com/user-attachments/assets/84d09b1e-6def-4873-8536-1b735f31413c">
then you will have the option to add the font to your class usring `@extend %t-poppins-32-medium;`
For now, you can do without variables. Just remember for the future

Reply:
Thank's for the advice) But for this project it would be unnecessary because I have tons of different mixes of: font-size, line-height and font-weight. For example first element should have fs: 12px, lh: 18px and fw: 400 while second element should have fs: 12px, lh: 16px and fw: 500 and they rarely have similar values. So I've decided only to split variables to different files without these changes and connect them in one.